src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -1030,17 +983,17 @@ private async ValueTask<int> ReadAsyncInternal<TIOAdapter>(TIOAdapter adapter, M
         private ValueTask FillHandshakeBufferAsync<TIOAdapter>(TIOAdapter adapter, int minSize)
```

Comment:
We should not need this, right? Since there is only one buffer now `EnsureFullTlsFrameAsync` should do it.

Reply:
Removed